muxfs/mux.go

```diff
@@ -12,15 +12,15 @@ import (
 	"github.com/qri-io/qfs/qipfs"
 )
 
-// NewMux creates a new path muxer
-func NewMux(handlers map[string]qfs.Filesystem) *Mux {
-	return &Mux{handlers: handlers}
-}
-
 // Mux multiplexes together multiple filesystems using path multiplexing.
 // It's a way to use multiple filesystem implementations as a single FS
 type Mux struct {
 	handlers map[string]qfs.Filesystem
+	// sophisticated writes require a legacy cafs.Filesystem interface
+	// the first configured filesystem that implements cafs.Filesystem
+	// will be set to this string, and returned by the CAFS method, creating a
+	// default write destination
+	defaultWriteDestination string
```

Comment:
🙌 

Think you must have written this comment before refactoring the CAFS method to `DefaultWriteFS`, should update comment.